doc/api/deprecations.md

```diff
@@ -2786,6 +2786,21 @@ These properties are now available within the standard `detail` property
 of the `PerformanceEntry` object. The existing accessors have been
 deprecated and should no longer be used.
 
+### DEP0153: `dns.lookup` and `dnsPromises.lookup` options type coercion
```

Comment:
I'm landing this now, but in general deprecation PRs should leave the code number unassigned (e.g. use `DEP0XXX`) so that the code can be assigned when it is landed.